src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -450,6 +450,19 @@
     <Field Name="Method" Type="MethodSymbol"/>
   </Node>
 
+  <!-- Represents the greatest raw metadata token value for any method definition in the current module. -->
```

Comment:
Half way through the changes I understood what this node is about. Consider renaming the node to MaxMethodDefIndex.